deploy/helm/sumologic/values.yaml

```diff
@@ -198,6 +198,9 @@ sumologic:
   ## Set the enabled flag to false for disabling metrics ingestion altogether.
   metrics:
     enabled: true
+    ## Set provider service (either fluentd or otelcol).
+    ## Otelcol is experimental and may behave in unexpected way.
```

Comment:
It seems that not all new parameters are added to deploy/helm/sumologic/README.md

Reply:
Fixed